src/arch/common/cc-clang.sh

```diff
@@ -19,6 +19,7 @@ if [ "$CMK_COMPILER" = "msvc" ]; then
   CMK_NATIVE_LDXX="$CMK_LDXX"
 fi
 
+CMK_PIC='' # reset to default
```

Comment:
could add here (and below) how this is reset. I believe it is in charmc?

Reply:
Done